interfaces/builtin/basedeclaration.go

```diff
@@ -154,6 +154,8 @@ plugs:
   snapd-control:
     allow-installation: false
     deny-auto-connection: true
+  unity8:
+    allow-installation: false
```

Comment:
IRCCloud is down, so I'll respond here.  Can you expand please on what you want from me here?  As I mentioned, I have seen TestPlugInstallation and have had a unity8 installability check there since the start.